src/ziggurat/mapper.clj

```diff
@@ -80,7 +80,7 @@
             (sentry/report-error sentry-reporter e (str "Channel execution failed for " topic-entity-name " and for channel " channel-name))
             (metrics/increment-count default-namespace failure-metric additional-tags)))))))
 
-(defrecord MessagePayload [message topic-entity])
+(defrecord MessagePayload [message topic-entity headers])
 
 (s/defschema message-payload-schema
```

Comment:
since you are adding headers to messagePayload, the schema will also change

Reply:
the schema is used to validate the message at the consumer end right? we are not passing headers anywhere in the consumers. thats why did not add it here